readme/vaccination.md

```diff
@@ -0,0 +1,19 @@
+## Vaccination
+This report searches different lab tests. It is a MSPP Report requested by the Haitian MoH to any clinic in Haiti.
```

Comment:
>This report searches different lab tests
?

Could you also provide more details of what it really expects in terms of data. For instance, from Catherine description:
"All children that have received 1 dose of BCG, 1 dose of VPI, 3 doses of Penta, 2 doses of VPO, 2 doses of Rota and 1 dose of RR"

Reply:
Done